### DISCUSSION TITLE
Unable to complete the student verification for strange reason

### DISCUSSION THREAD

**Discussion by touchsky-real** [community/community#55266] [GitHub Education] [answered]
### Hi everyone,

I am a student from China. I tried to complete the student verification in GitHub but got the same error after many attempts. 

error message:
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

My proof material is my acceptance letter. And I had added my real Chinese name to my profile since that's what appears in my acceptance letter. a bio also was added to my profile.
But it still doesn't work, so I have no idea of what's going on. Could someone assist me with this?

**Comment by jge162** [comment, marked as answer]
You should put your name in english translation, and include first and last name. Also try using a school ID or Transcripts to get verified. and write a little bit longer of a bio. @touchsky-real good luck.

**Reply by touchsky-real** [reply]
Thank you! I will try these methods 😊

**Comment by Ntshangase** [comment]
Hi @touchsky-real 

You need to provide a translation to your document.
For help writing translations checkout:  [Link](https://www.luogu.com.cn/blog/pnpn111/github-student-packet)

**Reply by touchsky-real** [reply]
That's very helpful! Thank you!

**Comment by Cretenet** [comment]
Hi everyone,
I have the same problem. I have an admission letter with a date, my name AND the uni logo. I put an english translation next to it and the system still refuses my application. I used the school email and my name is the one on the letter. Can someone help me?
